src/base_types.ads

```diff
@@ -194,4 +194,6 @@ package Base_Types is
    --  Returns the prefix common to Name1 and Name2 if any, or
    --  Null_Unbounded_String
 
+   function Get_Full_Name (Elt : DOM.Core.Node) return String;
+
 end Base_Types;
```

Comment:
Let me ask you to change the name, avoiding the "Get_" prefix, eg just "Full_Name" by itself. We're (slowing) moving the code to follow that naming scheme, although we're nowhere nearly completely finished.

Reply:
Done.

Reply:
Thanks!